services/QuillLMS/client/app/bundles/Diagnostic/actions/concepts-feedback.ts

```diff
@@ -1,6 +1,8 @@
 import { push } from 'react-router-redux';
 import C from '../constants';
-import { ConceptFeedbackApi, } from '../libs/concept_feedback_api.ts';
+import { ConceptFeedbackApi } from '../../Shared';
+
+const api = new ConceptFeedbackApi('connect');
```

Comment:
Diagnostic was previously using "connect" as the key (I'm not really sure why though?)

Reply:
I actually wonder if Diagnostic is using these at all, or if this is an artifact, since we don't render feedback in Diagnostic activities? It might be worth doing a sweep to see if we can just delete this file altogether.

Reply:
I'm pretty sure you're right-- I'll do that when I go to refactor the actual action files